### PR TITLE
Fixes #16386 - Mark notification names for translation

### PR DESCRIPTION
**Pull request by bkearney**

### CODE REVIEW

**Comment by jlsherrill** [comment]
[test]

**Comment by ehelms** [comment]
[test]

File: db/seeds.d/106-mail_notifications.rb [hunk 1]
```diff
@@ -5,6 +5,12 @@
 #
 ::User.current = ::User.anonymous_api_admin
 
+# The notification names are used as humanized labels. These need to be
+# translated as well as the description
+N_('Host errata advisory')
+N_('Sync errata')
+N_('Promote errata')
+
```
Comment (jlsherrill) [inline db/seeds.d/106-mail_notifications.rb]: would it make sense to just do:
```
notifications.each{ |notif| N_(notif[:name].to_s.gsub('_', ' ').capitalize ) }
```

or would that not get picked up by the translations?

**Comment by ehelms** [comment]
@bkearney This PR is waiting on your feedback

File: db/seeds.d/106-mail_notifications.rb (hunk 1 above)
Comment (bkearney) [inline db/seeds.d/106-mail_notifications.rb]: @jlsherrill I have not had good luck with gettext parsing and assuming code execution. I believe it is better to call them out explicitly.

**Comment by bkearney** [comment]
@ehelms responded

**Pull request merged**